app/components/icons/library.tsx

```diff
@@ -1664,3 +1664,21 @@ export const AvailableIcon = (props: SVGProps<SVGSVGElement>) => (
     />
   </svg>
 );
+
+export const ChangeIcon = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
Didnt we already have the same icon? I think we use it in the sidebar menu or something